src/screens/AdminPortal/OrgList/modal/OrganizationModal.tsx

```
@@ -25,21 +25,25 @@
+ * createOrg={handleCreateOrg}
+ * t={translate}
+ * tCommon={translateCommon}
+ * userData={currentUser}
  * />
  * ```
  *
  * @returns The rendered organization modal.
  */
 import React, { type ChangeEvent } from 'react';
-import { Button, Col, Form, Row } from 'react-bootstrap';
+import Button from 'react-bootstrap/Button';
```

ESLint

[error] 'react-bootstrap/Button' import is restricted from being used. Direct imports of react-bootstrap/Button are not allowed. Use the shared Button component from src/shared-components/Button/ instead.

(no-restricted-imports)